### DISCUSSION TITLE
track connected user

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#3565] [Q&A]
is there any way to track how many user are connected to realtime subscribe using hook..like i want if i use socket than we can track how many conneccted user if the user left also reduce the connected user..is it posssible to do using pocketbase?

**Comment by ganigeorgiev** [comment]
Please check [#3437](https://github.com/pocketbase/pocketbase/discussions/3437#discussioncomment-7173164) and https://github.com/pocketbase/pocketbase/discussions/3378.

**Comment by apujayed** [comment]
// note: we listen to the subscribe request and not to the "on connect" 
// because the authentication happens during the subscription submission
onRealtimeAfterSubscribeRequest((e) => {
    const record = e.client.get("authRecord")

    if (record && !record.get("online")) {
        record.set("online", true);
        $app.dao().saveRecord(record);
    }
})

onRealtimeDisconnectRequest((e) => {
    const record = e.client.get("authRecord")

    if (record && record.get("online")) {
        record.set("online", false);
        $app.dao().saveRecord(record);
    }
})``

but i want suppose i have a users collection in there have a coloumn status  how can i target specific user status true or  false?
how to get which user are subscribe?
